signingscript/test/test_task.py

```diff
@@ -1,25 +1,46 @@
-import pytest
+import unittest
+
+from scriptworker.context import Context
 from scriptworker.exceptions import ScriptWorkerTaskException
-from signingscript.task import task_signing_formats, task_cert_type
 
+from signingscript.task import task_signing_formats, task_cert_type, validate_task_schema
+from signingscript.script import get_default_config
+
+from signingscript.test.helpers.task_generator import TaskGenerator
+
+
+class TaskTest(unittest.TestCase):
+    def test_task_signing_formats(self):
+        task = {"scopes": ["project:releng:signing:cert:dep",
+                           "project:releng:signing:format:mar",
+                           "project:releng:signing:format:gpg"]}
+        self.assertEqual(["mar", "gpg"], task_signing_formats(task))
```

Comment:
Two things:

1. I know this is my bug from previous tests, but the cert scopes have changed since I've written these, e.g. `project:releng:signing:cert:dep-signing`.
2. with pytest this is `assert ["mar", "gpg"] == task_signing_formats(task)`.